pages/linux/ffuf.md

```diff
@@ -0,0 +1,28 @@
+# ffuf
+
+> Subdomain and directory discovery tool.
+> More information: <https://github.com/ffuf/ffuf>.
+
+- Discover directories using a [w]ordlist on a target [u]rl with [c]olorized and [v]erbose ouput:
+
+`ffuf -w {{path/to/wordlist}} -u {{https://target/FUZZ}} -c -v`
+
+- Fuzz [H]ost-headers with host file on a traget website and [m]atch HTTP 200 [c]ode responses:
+
+`ffuf -w {{hosts.txt}} -u {{https://example.org}} -H "{{Host: FUZZ}}" -mc {{200}}`
+
+- Discover directory using a [w]ordlist on a target website with a max individual job time of 60 seconds and recusrion discovery depth of 2 levels:
+
+`ffuf -w {{path/to/wordlist}} -u {{https://target/FUZZ}} -maxtime-job {{60}} -recursion -recursion-depth {{2}}`
+
+- Fuzz GET parameter on a target website and [f]ilter out message [s]ize response of 4242 bytes:
+
+`ffuf -w {{path/to/param_names.txt}} -u {{https://target/script.php?FUZZ=test_value}} -fs {{4242}}`
+
+- Fuzz POST method with POST [d]ata of password on a target website and [f]ilter out HTTP response [c]ode 401:
+
+`ffuf -w {{path/to/postdata.txt}} -X {{POST}} -d "{{username=admin\&password=FUZZ}}" -u {{https://target/login.php}} -fc {{401}}`
+
+- Discover subdomains using a subdoomain list on a target website:
+
+`ffuf -w {{subdomains.txt}} -u {{http://website.com}} -H "{{Host: FUZZ.website.com}}"`
```

Comment:
```suggestion
`ffuf -w {{subdomains.txt}} -u {{https://website.com}} -H "{{Host: FUZZ.website.com}}"`
```
Since we're using it everywhere else

Reply:
This brings up an interesting point.  HTTP vs. HTTPS.
The program can do both protocols non-secure and secure.  I went with examples using non-secure, as it is the simpler of the two.  That said, if we use HTTP/S in the examples, should there be a call out in the description of the program to inform that it is not protocol specific?
For example:
```
> Subdomain and directory discovery tool that uses HTTP or HTTP.
```
Thoughts?

Reply:
I think if we specify `HTTPS` it's clear that the program can also handle `HTTP`, but maybe not otherwise.

Reply:
Can we just drop it entirely?

```suggestion
`ffuf -w {{subdomains.txt}} -u {{website.com}} -H "{{Host: FUZZ.website.com}}"`
```

Reply:
The http/s is required for the program to work, other wise it will throw an error.

Reply:
Let's change it to `https` as @marchersimon suggests.